fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -226,25 +89,12 @@ private void subscribeWalletEvents() {
   @Override
   protected void onResume() {
     super.onResume();
-    mViewModel.openHome();
+    mViewModel.openHome(getSupportFragmentManager());
```

Comment:
It would be more impactful and, to me, go outside the scope of the PR.

We could do a PR to move the activity and fragment start in their respective class and then apply the livedata change.